repo/fsrepo.go

```diff
@@ -557,6 +557,11 @@ func (r *FSRepo) Path() (string, error) {
 	return r.path, nil
 }
 
+// JournalPath returns the path the journal is at.
+func (r *FSRepo) JournalPath() string {
+	return fmt.Sprintf("%s/filecoin_journal.json", r.path)
+}
+
```

Comment:
This should probably be configurable, but doing so now feels out of scope. I propose that if we want a configurable path we do it in a follow on.

Reply:
Non-configurable is fine. But drop the redundant `filecoin` in the file name.